Information_keys_overview (#28)
Meeco/docs: information/Keys.md

## Introduction

* What are keys?

* Why are (encryption) keys important in this industry?

> Meeco’s encryption key management service enables users to generate, manage and deploy keys to encrypt and protect data at rest, in transit and when exchanged. Our data and key architecture is designed for non-correlation, with all attributes encrypted with a Universally Unique Identifier (UUID). Provisioning UUIDs provides the foundation for Web3 applications where business rules and value drivers can be aligned to single identifiers.

Reviewer: ```suggestion
The key management service enables users to store and retrieve keys used to encrypt and sign data and other keys at rest, in transit and when exchanged. Cryptographic keys are an essential primitive used in operations such as, encrypting personal data, controlling a DID or signing a Verifiable Credential, to name a few.

As a rule of thumb, ALL encryption and signing happens on the client side and keys are _never_ visible to the platform. Therefore, users can choose to use this component or bring their own (existing) key management solution.
```

> ## Overview ##

Reviewer: ```suggestion
## Overview
```

> With a strong focus on privacy and security around the management and exchange of data, Meeco understands the importance of data encryption. With this in mind, we provide a key management service called the Keystore that enables users to easily access their data whilst ensuring it remains secure and protected. To do this we employ multiple cryptographic key types, all of which serve a purpose:

Reviewer: Let's not introduce "Keystore" as a term.

Author: Renamed everywhere "Keystore" mention to KMS (Key management service

Master Encryption Key (MEK): a derived key (created from a passphrase) that encrypts the KEK. It is held by each endpoint (user), and it is never shared.

Key Encryption Key (KEK): encrypts all other keys.

> Keypair: controls a DID.

Reviewer: ```suggestion
**Keypair** – used to ensure integrity (signing/verifying) and exchanging keys between users
```

Data Encryption Key (DEK): used to encrypt data that goes into the Vault (i.e. Items).There are two types of DEKs:​

DEK for personal data is used to encrypt and decrypt data exchanged with the Vault.​

> DEK for sharing data is used to encrypt and decrypt data exchanged with another user.

Reviewer: Can we make a list of this one. Format it as

- **Type of key** – explanation

Author: done
